examples/coap-server.c

```diff
@@ -517,15 +557,33 @@ main(int argc, char **argv) {
 
   clock_offset = time(NULL);
 
-  while ((opt = getopt(argc, argv, "A:g:p:v:l:")) != -1) {
+  while ((opt = getopt(argc, argv, "A:c:g:h:k:p:v:l:")) != -1) {
     switch (opt) {
     case 'A' :
       strncpy(addr_str, optarg, NI_MAXHOST-1);
       addr_str[NI_MAXHOST - 1] = '\0';
       break;
+    case 'c' :
+      cert_file = optarg;
+      break;
     case 'g' :
       group = optarg;
       break;
+    case 'h' :
```

Comment:
I see why this is `-h` but why not calling it `-u` in analogy to the client?

Reply:
In the client, it is the Identity, in the Server it is the Hint.  As -u is neither I or H, I guess it can be used without the confusion as to what it may be.

Reply:
"Username/password" is a concept many users should be familiar with. Maybe even more than psk_identity vs. psk_identity_hint.

Reply:
Well ok, in this case, it would have needed to be "realm", so just leave it `-h`.